### PR TITLE
Ghosts can no longer bully posibrains

### LINKED ISSUE #11486: Inhabited posibrains can still be pinged
(WEB REPORT BY: sprok REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 4d935af35fa01c86e8925136514deebb4f59b34d 
> 
> General description of the issue
> Posibrains with a player inside can still be pinged by ghosts
> 
> What you expected to happen
> That not to happen
> 
> What actually happened
> It does, get flushed/spaced
> 
> Steps to reproduce if possible
> Be brain, get pinged, everyone hates you

### PR DESCRIPTION
**Pull request by ghost**
fixes #11486

No more posibrain bully

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 1]
```diff
@@ -171,9 +171,10 @@
 	if(searching)
 		volunteer(O)
 	else
-		var/turf/T = get_turf(src.loc)
-		for (var/mob/M in viewers(T))
-			M.show_message("<span class='notice'>\The [src] pings softly.</span>")
+		if(!brainmob.ckey)
+			var/turf/T = get_turf(src.loc)
+			for (var/mob/M in viewers(T))
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/brain/posibrain.dm]: Change this to a visible message, please.

**Comment by Intigracy** [comment]
This also removes it pinging through walls.

Not sure if it was intended or not.

**Comment by Exxion** [comment]
Did it do that before? Because it checked `viewers()`

**Comment by Intigracy** [comment]
Yep.

**Comment by Exxion** [comment]
That doesn't make sense unless the person hearing the ping through the wall is wearing mesons

**Comment by Intigracy** [comment]
Maybe I was wearing mesons, I was a miner.

I just specifically remember hearing it ping from R&D while the posibrain was in robotics.

**Pull request merged**